python/llm/README.md

```diff
@@ -144,3 +141,5 @@ To avoid difficaulties during the installtion. `bigdl-llm` release the C impleme
 | gptneox      | Windows  | MSVC 19.36.32532.0 |       |
 | bloom        | Linux    | GCC 9.4.0          | 2.31  |
 | bloom        | Windows  | MSVC 19.36.32532.0 |       |
+| starcoder    | Linux    | GCC 9.4.0          | 2.31  |
+| starcoder    | Windows  | MSVC 19.36.32532.0 |       |
```

Comment:
some building parameters has been forked. We may need to rearrange this by split "convert", "inference" requirement for each family @Oscilloscope98

Reply:
please change 2.31->2.29 for both bloom and starcoder. We will use https://github.com/intel-analytics/BigDL/pull/8399 for a more detailed table later.

Reply:
For gptneox and llama, Linux GCC should be updated to 9.3.1